src/main/java/io/jenkins/infra/repository_permissions_updater/github_team_sync/GithubTeamDefinition.java

```diff
@@ -0,0 +1,44 @@
+package io.jenkins.infra.repository_permissions_updater.github_team_sync;
+
+import java.util.Set;
+
+public class GithubTeamDefinition {
+
+    private String RepoName = "";
+    private String TeamName = "";
+    private Set<String> developers;
+
+    public GithubTeamDefinition(String RepoName, String TeamName, Set<String> developers) {
+        this.RepoName = RepoName;
+        this.TeamName = TeamName;
+        this.developers = developers;
+    }
+
+    public GithubTeamDefinition() {
+    }
+
+    public String getName() {
+        return RepoName;
+    }
+
+    public void setName(String name) {
+        this.RepoName = name;
+    }
```

Comment:
why do you need these setters when you have a constructor which specifies all fields?

Reply:
Thank you for the suggestion! I'll remove all unnecessary setters.